dynatrace/configuration_v1/management_zones.py

```diff
@@ -0,0 +1,140 @@
+        self.negate: bool = raw_element.get("negate")
+        self.type: ComparisonBasicType = ComparisonBasicType(
+            raw_element.get("type"))
+        self.case_sensitive: bool = raw_element.get("caseSensitive")
+
+
+class ConditionKey(DynatraceObject):
+    def _create_from_raw_data(self, raw_element: Dict[str, Any]):
+        self.attribute: ConditionKeyAttribute = ConditionKeyAttribute(
+            raw_element.get("attribute"))
+        self.type: ConditionKeyType = ConditionKeyType(raw_element.get("type"))
+
+
+class EntityRuleEngineCondition(DynatraceObject):
+    def _create_from_raw_data(self, raw_element: Dict[str, Any]):
+        self.key: ConditionKey = ConditionKey(
+            raw_element=raw_element.get("key"))
+        self.comparison_info: ComparisonBasic = ComparisonBasic(
+            raw_element=raw_element.get("comparisonInfo"))
+
+
+class EntitySelectorBasedManagementZoneRule(DynatraceObject):
+    def _create_from_raw_data(self, raw_element: Dict[str, Any]):
+        self.enabled: bool = raw_element.get("enabled")
+        self.entity_selector: str = raw_element.get("entitySelector")
+        self.value_format: str = raw_element.get("valueFormat")
+
+
+class ManagementZoneRule(DynatraceObject):
+    def _create_from_raw_data(self, raw_element: Dict[str, Any]):
+        self.type: RuleType = RuleType(raw_element.get("type"))
+        self.enabled: bool = raw_element.get("enabled")
+        self.value_format: str = raw_element.get("valueFormat")
+        self.propagation_types: List[PropagationType] = [PropagationType(
+            prop_type) for prop_type in (raw_element.get("propagationTypes") or [])]
+        self.conditions: List[EntityRuleEngineCondition] = [
+            EntityRuleEngineCondition(raw_element=condition) for condition in (raw_element.get("conditions") or [])
+        ]
+
+
+class ManagementZone(DynatraceObject):
+    def _create_from_raw_data(self, raw_element: Dict[str, Any]):
+        self.metadata: ConfigurationMetadata = ConfigurationMetadata(
+            self._http_client, None, raw_element.get("metadata"))
+        self.id: str = raw_element.get("id")
+        self.name: str = raw_element.get("name")
+        self.description: str = raw_element.get("description")
+        self.rules: List[ManagementZoneRule] = [ManagementZoneRule(
+            raw_element=rule) for rule in raw_element.get("rules")]
+        self.entity_selector_based_rules: List[EntitySelectorBasedManagementZoneRule] = [
+            EntitySelectorBasedManagementZoneRule(raw_element=rule) for rule in (raw_element.get("entitySelectorBasedRules") or [])
+        ]
+
+
+class ManagementZoneShortRepresentation(EntityShortRepresentation):
+    def get_full_configuration(self):
+        """
+        Get the full configuration for this management zone short representation.
+        """
+        response = self._http_client.make_request(
+            f"{ManagementZoneService.ENDPOINT}_{self.id}").json()
```

Comment:
This string should be:

` f"{ManagementZoneService.ENDPOINT}/{self.id}`

currently with the underscore it will 404.